code/http4s/src/main/scala/com/avast/datadog4s/extension/http4s/DatadogMetricsOps.scala

```diff
@@ -18,7 +19,10 @@ object DatadogMetricsOps {
     implicit F: Sync[F]
   ): MetricsOps[F] = new MetricsOps[F] {
 
-    private[this] val activeRequests = metricFactory.count("active_requests")
+    private[this] val methodTag       = Tagger.make[String]("method")
+    private[this] val statusBucketTag = Tagger.make[String]("status_bucket")
```

Comment:
I'm not against grouping multiple statuses into single tag, yet I think we should still tag a metric with exact vaslue as well.

Reply:
yeah that is a mistake, i intended to have both